app/views/cases/search_filters/_filters.html.slim

```diff
@@ -8,35 +8,11 @@
         span.case-filters__summary.case-filters__summary--outer
           = t('filters.headings.filter_cases')
       .case-filters__container#filter-cases-accordion aria-hidden="true"
-        details
-          summary aria-controls="filter-status-content" aria-expanded="false" role="button"
-            span.case-filters__summary
-              = t('filters.headings.filter_by_status')
-          .case-filters__content#filter-status-content aria-hidden="true"
-            = render partial: 'cases/search_filters/open_case_status', locals: { f: f }
-        details
-          summary aria-controls="filter-type-content" aria-expanded="false" role="button"
-            span.case-filters__summary
-              = t('filters.headings.filter_case_type')
-          .case-filters__content aria-hidden="true"
-            = render partial: 'cases/search_filters/case_types', locals: { f: f }
-        details
-          summary aria-controls="filter-timeliness-content" aria-expanded="false" role="button"
-            span.case-filters__summary
-              = t('filters.headings.filter_by_timeliness')
-          .case-filters__content#filter-timeliness-content aria-hidden="true"
-            = render partial: 'cases/search_filters/timeliness', locals: { f: f }
-        details
-          summary aria-controls="filter-deadline-content" aria-expanded="false" role="button"
-            span.case-filters__summary
-              = t('filters.headings.filter_by_deadline')
-          .case-filters__content#filter-deadline-content aria-hidden="true"
-            = render partial: 'cases/search_filters/final_deadline', locals: { f: f }
-        - if request.path.include? '/cases/search'
+        - @query.available_filters(current_user, @current_tab_name).each do | filter |
           details
-            summary aria-controls="filter-exemptions-content" aria-expanded="false" role="button"
+            summary aria-controls="filter-#{}-content" aria-expanded="false" role="button"
```

Comment:
Is something meant to be interpolated here in `filter-#{}-content`? looks like an object to interpolate is missing to me 🤔

Reply:
mistake, didn't cause any issue for running the app, but caused the failure of test spec